README.md

```diff
@@ -1,24 +1,4 @@
 # README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+To run the application, 
+run "./bin/dev" and in the terminal they mention the port link, 
+use that link to run the application on localhost
```

Comment:
That's a great start for us! But I think let's add a few more details to make it more readable for others, by adding all of these: https://github.com/girijaasoni/newsletterApplicaton/issues/7 (I'm on it!)